README.md

```diff
@@ -18,15 +18,27 @@ This project contains a number of components:
 * [AWS CDK Infrastructure as Code](./infrastructure/README.md)
 * [GitHub Actions CI/CD pipeline](./.github)
 
-## Development setup
+## Running Locally
+To run the application locally you need 3 components running.
+* An Algorand LocalNet
+* The [Write-through cache Algorand IPFS gateway](./src/voting-metadata-api/README.md)
+* The [Voting dApp](./src/dapp/README.md)
 
+Here are the steps you can follow to run the application locally:
 1. Install `AlgoKit` - [Link](https://github.com/algorandfoundation/algokit-cli#install): Ensure you can execute `algokit --version`.
 2. Run `algokit localnet start` to start a LocalNet network
-3. Run `algokit bootstrap all` in `src` to install dependencies and set up `.env` files
```

Comment:
The instructions on the left are correct and probably faster, plus mention the VS Code F5 option, should we keep  or augment them?

Reply:
From the existing, I get errors when I run step 3 and step 4 is unnecessary to run the application locally

Reply:
It's not if running via VS Code, but I will change that.